src/middlewares/version.ts

```diff
@@ -43,11 +45,7 @@ export const checkVersion = async (_yargs: yargs.ArgumentsCamelCase) => {
     // Write the version-check file first to avoid retrying errors
     await fs.writeFile(latestFile, "");
 
-    // Note that package.json is installed one level above "dist"
-    // We can't require package.json as it is outside the TypeScript root
-    const { name, version } = JSON.parse(
-      (await fs.readFile(`${__dirname}/../../package.json`)).toString("utf-8")
```

Comment:
I think this may have been broken since 0.17.1. Users on 0.17.1 to the current version may not ever see the upgrade message, since this is reading the file at the wrong location.

They may be "stuck" where they are since they have no way to know that the version check failed (the catch block below doesn't do anything).